ptyprocess/ptyprocess.py

```diff
@@ -206,6 +206,19 @@ def spawn(
         if not isinstance(argv, (list, tuple)):
             raise TypeError("Expected a list or tuple for argv, got %r" % argv)
 
+        # env vars must be utf-8 encoded strings
```

Comment:
Always UTF-8? Or should we be trying to apply a locale-dependent encoding?

Reply:
On reflection, unless this comment makes you go "aha, yes, I know just how we should do this!", let's stick to UTF-8 encoding and see if anyone complains. It might not be 100% right, but it's easy to understand, and people who really need something else can still pass environment variables as bytes.